examples/asr/conf/config.yaml

```diff
@@ -1,11 +1,40 @@
 #model: "QuartzNet15x5"
 sample_rate: &sample_rate 16000
-accumulate_grad_batches: &accumulate_grad_batches 1
+repeat: &repeat 1
+dropout: &dropout 0.0
+separable: &separable true
 
-optim_params:
-  name: adam 
+optim:
+  name: novograd
   lr: .01
-  args: {}
+  # optimizer arguments
+  args:
+    name: auto
+    # cls: nemo.core.config.optimizers.NovogradParams
+    params:
+      betas: [0.8, 0.5]
+      weight_decay: 0.001
+
+  # scheduler setup
+  sched:
+    name: CosineAnnealing
+    iters_per_batch: null # computed at runtime
```

Comment:
It is my understanding that "iters_per_batch" is always computed at runtime. Do we have to include it in the .yaml file and Config object?